radicle-node/src/address/types.rs

```diff
@@ -1,8 +1,8 @@
 use std::ops::{Deref, DerefMut};
 
 use nonempty::NonEmpty;
-use radicle::node;
-use radicle::node::Address;
+
+use radicle::node::{self, Address};
```

Comment:
I try to avoid using `self` in imports.

Reply:
Agreed, but we've went with `self` elsewhere, so I just wanted to stick to the same formatting.